<div class="mb-8">
    <img src="{{ url_for('static', filename='img/0001/tmem81-by-af.jpg') }}"
         alt="Banner" class="w-full h-96 object-cover rounded">
    <p class="text-xs text-gray-500 mt-2">Credit: Google DeepMind/EMBL-EBI (CC-BY-4.0)</p>
</div>

<p>
    Above is an AlphaFold interpretation of the protein Tmem81 structure,
    used as a banner for the
    <a href="https://www.nature.com/articles/d41586-025-03886-9"  target="_blank" >Nature editorial from Nov 26, 2025</a>,
    celebrating the fifth anniversary of AlphaFold.
    The main protagonist of the story is prof. Andrea Pauli, whose several-year project was finally
    <a href="https://www.cell.com/cell/fulltext/S0092-8674(24)01093-6"  target="_blank"> brought to shore </a>
    thanks to
    <a href="https://www.biorxiv.org/content/10.1101/2021.10.04.463034v2" target="_blank">AlphaFold-Multimer</a>
    (I could not locate a reviewed version of the related publication).

</p>

<!--------------------------------------------------------------------------------------------->
<div class="flex gap-6 ">
    <div class="w-1/4">
        <img src="{{ url_for('static', filename='img/0001/7ta2.png') }}"
             alt="Side image of 7ta2 structure" class="max-w-full">
    </div>
    <div class="w-3/4">
        <p>
            In their work,  Pauli's lab established that Tmem81 forms trimers with Izumo1 and Spaca6.
        </p>

        <h3 class="text-2xl font-semibold mt-6 mb-2">SPACA6</h3>
        <p>
            The structure of SPACA6 was published in 2021, PDB id
            <a href="https://www.rcsb.org/structure/7TA2" target="_blank">7TA2</a>.
            It is shown on the left. According to the
            <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC9482655/#Sec12:~:text=At%20the%20time%20of%20structure%20determination%2C%20there%20were%20no%20experimental%20or%20AlphaFold%20structures%20available%2E"
               target="_blank"> accompanying paper,
            </a>
            at the time of structure determination, there were no experimental or
            AlphaFold structures available. Shown in the figure below is the superposition of 7TA2 structure (cyan), with some ions (red), and the
            AF2 structure prediction (magenta, the structure with the corkscrew thing), as of Aug 2025.

        </p>

        <span class="inline-block max-w-full -mt-1280">
            <img src="{{ url_for('static', filename='img/0001/W5XKT8-F1_supp_7ta2.png') }}"
                 alt="Page width image 7ta2 structure superimposed ofn AF2 prediction" class="max-w-full">
        </span>

    </div>
</div>


<!--------------------------------------------------------------------------------------------->
<div class="flex gap-6 ">
    <div class="w-3/4">
        <h3 class="text-2xl font-semibold mt-6 mb-2">IZUMO1</h3>
        <p>
            Two groups published, back to back in Nature 534, 2016, the structure of IZUMO1,
            <a href="https://www.rcsb.org/structure/5JK9" target="_blank">5JK9</a>, by Ohto et al., and
            <a href="https://www.rcsb.org/structure/5F4T" target="_blank">5F4T</a>, by Aydin et al.

            The superposition of these two structures is shown on the right.
            Below is the superposition of the AF2 prediction (magenta) on the two crystallized structures (green, cyan)
        </p>

        <span class="inline-block max-w-full -mt-1280">
            <img src="{{ url_for('static', filename='img/0001/Q8IYV9-F1_supp_izumo1.png') }}"
                 alt="Page width image 5JK9 and 5F4T structures superimposed on AF2 prediction" class="max-w-full">
        </span>
    </div>

    <div class="w-1/4">
        <img src="{{ url_for('static', filename='img/0001/izumo1_overlap.png') }}"
             alt="The close overlap of the two Izumo1 structures ( 5JK9 and 5F4T) published in 2016" class="max-w-full">
    </div>

</div>



<!--------------------------------------------------------------------------------------------->
<div class="flex gap-6">
    <div class="w-full ">
        <h3 class="text-2xl font-semibold mt-6 mb-2">TMEM81</h3>
        <p>
            Back to the main protagonist — TMEM81. In distinction to modeling its cousins IZUMO1 and SPACA6, here AF2 is
            on its own. So it predicts some helices, a small beta strand and an immunoglobulin-like domain.
            Their relative 3D arrangement is left to our imagination.
        </p>
        <br>
        <p>
            But how does AF do as much as that - getting the immunoglobulin-like domain right?
            After all, marvels a publication contemporary to the one form the Prof. Pauli's group,
            the sequence identity to the (Ig)-like domains in  IZUMO1 and SPACA6
            <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11052572/#:~:text=16%E2%80%9327" target="_blank">is 16–27% </a>.
        </p>
        <br>
        <p>
            As it turns out, there are better ways to look for structural similarity, and if AF is worth its salt, it has
            figured it out.  You can look for a sequence of <em>secondary structure elements</em> corresponding to TMEM81,
            together with some judicious grouping of amino acids by their physical characteristics and alignment of the
            structurally important cysteines. There is one server that comes to my mind right away
            (there may be more) that does just that: <a href="https://aideepmed.com/I-TASSER/" target="_blank">I-TASSER</a>.
            The illustration below shows the idea. For more you will have to run I-TASSER yourself, they do not
            store the results for very long:
        </p>
        <br>
        <div class="inline-block max-w-full  min-w-[800px] -mt-1280">
            <img src="{{ url_for('static', filename='img/0001/i-tasser-pred.png') }}"
                 alt="Illustration of I-Tasser finding and reporting the similarity in the (predicted) secondary structure"
                 class="max-w-full space-x-4">
        </div>
        <p>
            Among else, note the honestly reported crystal structures that the model is based on.
            Here is also a zoom into  the IG-region:
        </p>
        <div class="inline-block max-w-full  min-w-[500px] -mt-1280">
            <img src="{{ url_for('static', filename='img/0001/i-tasser-ig-domain.png') }}"
                 alt="Illustration of I-Tasser finding and reporting the similarity in the (predicted) secondary structure"
                 class="max-w-full space-x-4"/>
        </div>

        <br/>
        <p>
            There is even some sequence similarity here that can be forced out of
            <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">BLAST</a>
            search by adapting the parameters to
            short sequence search (I tried).


        </p>
        <p>
            The similarity is high enough for PyMol to align the TMEM81 ig-like
            domain to its counterpart in IZUMO1 (try as homework).
            N.B: Pymol's default algorithm
            <a href="https://pymolwiki.org/Align" target="_blank">aligns by sequence,</a>
            not by structural similarity.
            Here is a somewhat  messy rendition of the PyMol superposition of the IG-like domains in the SPACA6 / IZUMO1 /TMEM81
            trio. The highlighted pieces of the sequence are enough to guide the alignment.
        </p>
        <div class="inline-block max-w-full  min-w-[600px] -mt-1280">
            <img src="{{ url_for('static', filename='img/0001/pymol_ig_superposition.png') }}"
                 alt="Illustration of PyMol structurla alignment of the IG-like domains in the SPACA6 / IZUMO1 /TMEM81"
                 class="max-w-full space-x-4">
        </div>



        <!--- there is som e intersting substory here - Izumo1 was noted to be similar to plasmodium invasion proteins
        (https://pubmed.ncbi.nlm.nih.gov/27374339/) and poliovirus receptor (CD155)  - the latter actually being
        a human protein that the poli viurs exploits - CD155 (cluster of differentiation 155), a
        lso known as the poliovirus receptor, is a protein that in humans is encoded by the PVR gene.[3][4]
        It is a transmembrane protein that is involved in forming junctions between neighboring cells.
         It is also the molecule that poliovirus uses to enter cells. The gene is specific to the primates.
         That's the 1dgiR structure reported by I-TASSER
        -->
        <p>
            Bottom line: up to this point AF2 did not "predict" or "fold" anything new that could not have been done
            with more pedestrian methods. An odd choice to celebrate its fifth birthday.
        </p>
        <br/>
        <p>
            Coda: To be fair to prof Pauli, his enthusiasm came from the trimerization structure prediction by AlphaFold-Multimer,
            not from AF3 itself. To be noted, however, his lab's work did not prove that the predicted structure is correct,
            but that the three proteins very
            likely form a trimer. So how did AF get that? AF-Multimer is trained on known protein complexes.
            Ig-like domains are popular among adhesion proteins. Let's return to this question in another post.
        </p>
        <!--- check ed the the elif structure of the complex is the same. so at list the thing is consistent.
        Th e-life group marvers hoe the "template free" prediction looks the same as pn of the complexes it was trained on
        --->


    </div>

</div>
